cypress/e2e/felony/class-a/pages/f-classa-main-1-ineligble.cy.ts

```diff
@@ -0,0 +1,15 @@
+import testIneligiblePage from "../../utils/testIneligiblePage"
+
+describe("Felony Classification Page", () => {
+  it('should redirect to Felony Class A page when choosing Felony Class A option', () => {
+    cy.visit('/calculator/head-felony-4-cont');
+    cy.get('[data-cy="calc-choice-0"]')
+      .should('be.visible')
+      .should('contain', 'Felony Class A')
+      .click();
+    cy.url().should('include', 'calculator/f-classa-main-1-ineligble');
+  })
+  it('should display ineligibility content due to RCW 9.94A.030', () => {
+    testIneligiblePage('f-classa-main-1-ineligble', 'RCW 9.94A.030', 'https://app.leg.wa.gov/RCW/default.aspx?cite=9.94A.030');
+  })
```

Comment:
We have different contexts here.
The first `it` is testing `/calculator/head-felony-4-cont` page, and if this page send to `/calculator/f-classa-main-1-ineligble` page.

The second one is testing `/calculator/f-classa-main-1-ineligble` and if there is the `RCW` content on this.

And this file is called `f-classa-main-1-ineligble.cy.ts`, so make sense to test only things related to that page.

I would suggest to separate those contexts, creating another file called `head-felony-4-cont.cy.ts` to test things related to this page, and use the `f-classa-main-1-ineligble.cy.ts` file with only its related things.

Reply:
Sounds good! I refactored my tests to reflect this suggestion.